framework/src/fvbcs/FVFluxBC.C

```diff
@@ -174,7 +174,8 @@ FVFluxBC::computeJacobian(const FaceInfo & fi)
                  fi.faceCentroid());
   };
 
-  _assembly.processDerivatives(r, _var.dofIndices()[0], _matrix_tags, local_functor);
+  if (_var.dofIndices().size() > 0)
+    _assembly.processDerivatives(r, _var.dofIndices()[0], _matrix_tags, local_functor);
```

Comment:
I think I encountered this recently. If we don't have any dof indices here, then we are in big trouble. It means we're not applying a BC when we should be. What I suspect is true is that the dof indices live on the neighbor side of the face. In this case we should either be using data from the neighbor side and feeding the result into the "neighbor" residual (e.g. the row corresponding to `_var.dofIndicesNeighbor()[0]`) or we should be erroring out here. We shouldn't let this stand!

Reply:
I think that is the correct fix.
Thanks!